### PR TITLE
ADF-23 | Developer console update, env and help commands

### PR DESCRIPTION
**Pull request by alexmunoz502**
image

### CODE REVIEW

File: global/autoload/audio/audio.gd [hunk 1]
```diff
@@ -83,7 +83,7 @@ func _connect_signals() -> void:
 
 
 func _on_tab_changed(tab_data: TabData) -> void:
-	if tab_data.id == DarknessScreen.TAB_DATA_ID:
-		Audio.swap_crossfade_music(1)
-	else:
-		Audio.swap_crossfade_music(0)
+	# if tab_data.id == DarknessScreen.TAB_DATA_ID:
```
Comment (alexmunoz502) [inline global/autoload/audio/audio.gd:86]: this isn't part of this PR, but a work around for an unrelated issue

**Comment by alexmunoz502** [comment]
decided to change 'gamemode' to 'env' from the ticket in trello. LMK if you prefer gamemode instead

**Review by TinyTakinTeller — CHANGES_REQUESTED** [review]
Env makes sense for OS command.

For gamemode I imagined a different command - being able to change the params in console (“global/const/game.gd”) 👀 i.e. refactor that const node to a global node and allow to switch between the two dictionaries - what do you think about this idea?

**Comment by alexmunoz502** [comment]
> Env makes sense for OS command.
> 
> For gamemode I imagined a different command - being able to change the params in console (“global/const/game.gd”) 👀 i.e. refactor that const node to a global node and allow to switch between the two dictionaries - what do you think about this idea?
> 
> (If you think it is better to leave those as const i.e. only allow manual change in code before game startup, I'm also fine with that and will approve merge.)

Yeah, I think that it should just be set before running. We could have different environment presets that can be selected between on startup. Is there a use case where you think it might be necessary to edit constants?

**Review by TinyTakinTeller — APPROVED** [review]

**Pull request merged**